decidim-elections/spec/forms/decidim/elections/trustee_zone/trustee_form_spec.rb

```diff
@@ -40,4 +40,14 @@
 
     it { is_expected.not_to be_valid }
   end
+
+  context "when a trustee with the same name exists" do
+    let(:other_trustee) { create(:trustee, :with_public_key, name: "Shelton Runolfsson Sr.") }
+
+    before do
+      other_trustee
+    end
```

Comment:
If you want the variable to be evaluated right away (instead of waiting for it to be called) you can use `let!`

```suggestion
    let!(:other_trustee) { create(:trustee, :with_public_key, name: "Shelton Runolfsson Sr.") }
```